_posts/2020-04-29-monitoring-aws-with-panther.md

```diff
@@ -1,65 +0,0 @@
----
-layout: post
-title: "Using Panther to monitor AWS infrastructure"
-tags:
-- monitoring
-- aws
-- featured
-team: Security Engineering
-author: paha
----
-
-Before widespread cloud usage, it was uncommon for one person to be present for the entire datacenter development lifecycle. Very few people knew how to design and build a datacenter from scratch while ensuring appropriate security configuration settings were set, on top of rigging up monitoring. It was even more uncommon for non-sysadmins to have any involvement in data center infrastructure construction or ongoing refinement. The cloud is very different. It only takes seconds to create an entire infrastructure from a template. And even developers are doing it!
```

Comment:
These are public blog posts and I think there is benefit to this blog post staying online. I would rather a notice put at the top of the blog post stating that "Scribd's security infrastructure has since evolved away from using Panther"

Reply:
Good point @rtyler, I'll push the changes shortly

CC @johndotpub